pgp/keysource.go

```diff
@@ -169,8 +270,8 @@ func (key *MasterKey) fingerprintMap(ring openpgp.EntityList) map[string]openpgp
 func (key *MasterKey) passphrasePrompt(keys []openpgp.Key, symmetric bool) ([]byte, error) {
 	conn, err := gpgagent.NewConn()
 	if err == gpgagent.ErrNoAgent {
-		fmt.Println("gpg-agent not found, continuing with manual passphrase input...")
-		fmt.Print("Enter PGP key passphrase: ")
+		log.Printf("gpg-agent not found, continuing with manual passphrase input...")
+		log.Print("Enter PGP key passphrase: ")
```

Comment:
this isn't strictly speaking a log, it's a prompt

Reply:
Yes, I fixed that in my logging branch.